src/cpu/x64/matmul/brgemm_matmul_utils.cpp

```diff
@@ -1609,6 +1610,10 @@ status_t init_brgemm_matmul_conf(cpu_isa_t isa, brgemm_matmul_conf_t &bgmmc,
     if (merge_batch_dims_into_M
             && (src_d.matches_tag(acbd) || bgmmc.treat_transposed_A_as_plain)) {
         bgmmc.A_strides[1] = bgmmc.A_strides[2];
+
+        // Mark A_stride as changed
+        // In order to proper initialize LDA further
+        bgmmc.a_strides_changed = true;
```

Comment:
This will make the code more concise: 
```cpp
bgmmc.adjust_a_strides = merge_batch_dims_into_M && (src_d.matches_tag(acbd) || bgmmc.treat_transposed_A_as_plain);

if (bgmmc.adjust_a_strides) bgmmc.A_strides[1] = bgmmc.A_strides[2];
```

Reply:
Updated! Thanks!